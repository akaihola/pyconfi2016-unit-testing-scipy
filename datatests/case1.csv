time;signal1;signal2
30.10.16 12:00:00;14.0;55.5
30.10.16 12:01:00;5.0;-100.0
30.10.16 12:02:00;;42.0
30.10.16 12:03:00;;
30.10.16 12:04:00;0.001;1000000.0
