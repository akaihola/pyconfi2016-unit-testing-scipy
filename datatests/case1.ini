[METADATA]
format = 2

[TEST]

[result]
action = sum
inputs =
  [signal1]
  [signal2]
