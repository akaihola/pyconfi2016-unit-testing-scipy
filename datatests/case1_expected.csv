time;result
30.10.16 12:00:00;69.5
30.10.16 12:01:00;-95.0
30.10.16 12:02:00;
30.10.16 12:03:00;

30.10.16 12:04:00;1000042.0
